### PR TITLE
Devex pdf docket record

### PR DESCRIPTION
**Pull request by matthopson**
Printable Docket Record generated via React templates.

Screen Shot 2020-05-05 at 1 28 39 PM

### CODE REVIEW

**Review by sutt0n — APPROVED** [review]

File: shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  return (
+    <>
+      {contact.inCareOf && <div>c/o {contact.inCareOf}</div>}
+      {contact.title && <div>{contact.title}</div>}
+      {contact.address1 && <div>{contact.address1}</div>}
+      {contact.address2 && <div>{contact.address2}</div>}
+      {contact.address3 && <div>{contact.address3}</div>}
+      <div>
+        {contact.city && <span>{contact.city}, </span>}
+        {contact.state && <span>{contact.state}, </span>}
```
Comment (kkoskelin) [inline shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx]: nit: unsure that a comma is necessary after the state

Comment (matthopson) [reply shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx]: I was matching the original template. Perhaps @klohman could weigh in?

File: shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+};
+
+const DocketRecord = ({ caseDetail, countryTypes, entries, options }) => {
+  return (
+    <div id="document-docket-record">
+      <PrimaryHeader />
+      <CompressedDocketHeader
+        caseCaptionExtension={options.caseCaptionExtension}
+        caseTitle={options.caseTitle}
+        docketNumberWithSuffix={options.docketNumberWithSuffix}
```
Comment (kkoskelin) [inline shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx:85]: if you want to be extra-clever, I think you can just `...options` here.

File: shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx (hunk 1 above)
Comment (klohman) [reply shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx]: We don't use the comma after the state on the actual Case Detail screen. I would drop it from the PDF as well.

**Pull request merged**